test/plugins/features/movies/index.test.js

```diff
@@ -1,9 +1,51 @@
 'use strict';
 
-const Movies = require('../../../../lib/server');
+const Movies    = require('../../../../lib/server');
+const Constants = require('../../../helper');
 
 describe('movies integration', () => {
 
+  describe('filter', () => {
+
+    it('lists all movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        query: {}
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.length).to.eql(3);
+      });
+    });
+
+    it('filters by title', () => {
```

Comment:
Since you already unit test the filtering in the controller tests, you don't need these integration tests of filtering.